x/jwt_helper.go

```diff
@@ -56,19 +57,23 @@ func ExtractUserName(jwtToken string) (string, error) {
 	return userId, nil
 }
 
-func ExtractJWTNamespace(ctx context.Context) (uint64, error) {
-	jwtString, err := ExtractJwt(ctx)
-	if err != nil {
-		return 0, err
-	}
-	claims, err := ParseJWT(jwtString)
+func ExtractNamespaceFromJwt(jwtToken string) (uint64, error) {
+	claims, err := ParseJWT(jwtToken)
 	if err != nil {
-		return 0, err
+		return 0, errors.Wrap(err, "extracting namespace from JWT")
 	}
-
 	namespace, ok := claims["namespace"].(float64)
 	if !ok {
 		return 0, errors.Errorf("namespace in claims is not valid:%v", namespace)
 	}
 	return uint64(namespace), nil
 }
+
+func ExtractJWTNamespace(ctx context.Context) (uint64, error) {
```

Comment:
This function name doesn't seem to be very expressive of what the function is trying to. For example, I can't tell the difference between the functions from the name, "ExtractNamespaceFromJWT", and "ExtractJWTNamespace". Can you change the function names or add some comments around what the function is actually doing?